recipes/flang-activation/activate.sh

```diff
@@ -0,0 +1,8 @@
+export _OLD_FC=$FC
+export _OLD_FFLAGS=$FFLAGS
+export _OLD_FORTRANFLAGS=$FORTRANFLAGS
+
+export FORTRANFLAGS="$FORTRANFLAGS --sysroot=$CONDA_BUILD_SYSROOT"
+
+export FC="flang-new"
+export FFLAGS=$FORTRANFLAGS
```

Comment:
```suggestion
export FORTRANFLAGS="$FORTRANFLAGS"

export FC="flang-new"
export FFLAGS="$FFLAGS  --sysroot=$CONDA_BUILD_SYSROOT"
```

Reply:
With `gcc_linux-64`, CONDA_BUILD_SYSROOT is not set right?

Reply:
I don't really know the difference between `FFLAGS` and `FORTRANFLAGS`; we added the sysroot to both in https://github.com/conda-forge/flang-feedstock/commit/267195daf179af87ec0812b9c4c1164337b6aae9, which was when we [switched](https://github.com/conda-forge/flang-feedstock/pull/28) to llvm-flang.

To answer your question, AFAICT `gcc_linux-64` [does set](https://github.com/conda-forge/ctng-compiler-activation-feedstock/blob/e22827cd872402d807ae3452e23673d6d3e4b0b0/recipe/activate-gcc.sh#L187) `CONDA_BUILD_SYSROOT`

Reply:
Sorry, I meant without `gcc_linux-64` in run, we won't have `CONDA_BUILD_SYSROOT`, right?

Reply:
Yeah, it seems we're only setting (rather than using) that in `gcc_{{ target }}` and [`clang_{{ target }}`](https://github.com/conda-forge/clang-compiler-activation-feedstock/blob/52e3513c4592a367930372717c22cc5fb680d133/recipe/activate-clang.sh#L197).

Do you want us to remove the `--sysroot` flag here, or reconstruct `CONDA_BUILD_SYSROOT`?

Reply:
or depend on `gcc_linux64`...

Reply:
> Do you want us to remove the `--sysroot` flag here

At least the helloworld example on the flang feedstock passes without that it seems (no `gcc_linux-64` in environment, `CONDA_BUILD_SYSROOT` not set):

<details>

```
TEST START: /home/conda/feedstock_root/build_artifacts/linux-64/flang-19.1.0.rc3-h43464a6_0.conda
[...]
The following NEW packages will be INSTALLED:

    _libgcc_mutex:                        0.1-conda_forge               conda-forge              
    _openmp_mutex:                        4.5-2_gnu                     conda-forge              
    _sysroot_linux-64_curr_repodata_hack: 3-h69a702a_16                 conda-forge              
    binutils_impl_linux-64:               2.40-ha1999f0_7               conda-forge              
    bzip2:                                1.0.8-h4bc722e_7              conda-forge              
    clang:                                19.1.0.rc3-default_he6a287e_0 conda-forge/label/llvm_rc
    clang-19:                             19.1.0.rc3-default_hc83ec93_0 conda-forge/label/llvm_rc
    flang:                                19.1.0.rc3-h43464a6_0         local                    
    grep:                                 3.11-h7af0fdc_3               conda-forge              
    icu:                                  75.1-he02047a_0               conda-forge              
    kernel-headers_linux-64:              3.10.0-h4a8ded7_16            conda-forge              
    ld_impl_linux-64:                     2.40-hf3520f5_7               conda-forge              
    libclang-cpp19.1:                     19.1.0.rc3-default_hc83ec93_0 conda-forge/label/llvm_rc
    libflang:                             19.1.0.rc3-h43464a6_0         local                    
    libgcc-devel_linux-64:                14.1.0-h5d3d1c9_100           conda-forge
    [...]

Preparing transaction: ...working... done
Verifying transaction: ...working... done
Executing transaction: ...working... done
export PREFIX=/home/conda/feedstock_root/build_artifacts/flang-split_1724314217473/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac
export SRC_DIR=/home/conda/feedstock_root/build_artifacts/flang-split_1724314217473/test_tmp
+ flang-new --sysroot= hello_world.f90
+ ./a.out
 Hello World!
```

</details>

Reply:
I've removed that flag for now. If we do need it, we should IMO add a test in the activation-feedstock that exercise the necessity to use the sysroot.